app/policies/application_policy.rb

```diff
@@ -0,0 +1,28 @@
+class ApplicationPolicy
+  attr_reader :person, :record
+
+  def initialize(person, record)
+    @person = person
+    @record = record
+  end
+
+  def create?
+    @person.admin?
+  end
+
+  def new?
+    create?
+  end
+
+  def update?
+    @person.admin?
+  end
+
+  def edit?
+    update?
+  end
+
+  def destroy?
+    @person.admin?
+  end
+end
```

Comment:
From @brendandeere 

Since you are defining readers for these instance variables you might want to use them here instead of accessing the instance variable directly